drivers/sensor/tmag5273/Kconfig

```diff
@@ -0,0 +1,364 @@
+# TMAG5273 Low-Power Linear 3D Hall-Effect Sensor configuration options
+
+# Copyright (c) 2022 deveritec GmbH
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig TMAG5273
+	bool "TMAG5273 3D Hall-Effect Sensor"
+	depends on DT_HAS_TI_TMAG5273_ENABLED
+	select I2C
+	help
+		Enable driver for TMAG5273 I2C-based 3d Hall-Effect sensor.
+
+if TMAG5273
+
+comment "measurement"
+
+config TMAG5273_MAGNETIC_CH_X_EN
+	bool "Measure x-axis"
+	default y
```

Comment:
Normally we don't specify a default for bool options (which means the default is n).

Maybe it's different for accelerometer sensor channels - @MaureenHelm do you know if there the case?